### DISCUSSION TITLE
SEARCH HTTP method deprecation over QUERY

### DISCUSSION THREAD

**Discussion by tofran** [directus/directus#11362] [Feature Requests]
The latest revision of [IETF's HTTP search draft](https://datatracker.ietf.org/doc/draft-ietf-httpbis-safe-method-w-body/) uses the `QUERY` method, not `SEARCH` as [stated in Directus documentation](https://docs.directus.io/reference/introduction/#search-http-method).

As of Nov 2021 the `SEARCH` verb got replaced with `QUERY` with the draft revision `#02` ([draft-ietf-httpbis-safe-method-w-body-02](https://datatracker.ietf.org/doc/draft-ietf-httpbis-safe-method-w-body/02/)).

- Should Directus deprecate `SEARCH` in favor of `QUERY`?
- If so, in what release should this happen?
- Should both methods be supported while `SEARCH` is not removed?
   Can I make a pull request with this?

**Comment by rijkvanzanten** [comment]
Very good questions. Changing SEARCH to QUERY now will be a breaking change, so we won't deprecate the current SEARCH support any time soon. For now, I think we should wait it out a bit. Let the spec make it through the cycles, and then circle back once it's finalized and accepted.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
